cockroachcloud/authorization.md

```diff
@@ -1,44 +1,165 @@
+
+### Is it possible to assign more than 1 role to a user in a CockroachDB Cloud organization?
+
+Yes, it is possible to assign more than one role to a user. The default minimum access role Org Member is always assigned to every user as long as they’re a part of the CockroachDB Cloud organization. Beyond that, every other assigned role is additive to the overall entitlements of a user. Best example of this is the initial user who is by default assigned the Org Member, Org Admin, Cluster Admin and Billing Coordinator [to be added soon] roles when they create the CockroachDB Cloud organization. 
+
+### Can we follow the least privilege principle by using the roles available in the CockroachDB Cloud authorization model?
+
+Yes, the roles available in the CockroachDB Cloud authorization model allow admins to grant only those entitlements to users that are supposed to map to their intended workflows. 
+
+Cluster level roles like Cluster Admin, Cluster Operator Reader & Writer [to be added soon] or Cluster Developer allow to perform pertinent actions for one or more clusters, while providing differentiation between admin, operational hybrid and non-admin entitlements. Whereas, the Organization level roles like Org Admin and Org Member allow admin and non-admin access respectively for the entire organization, while avoiding overlap with cluster specific actions.
+
+We discourage further use of the Legacy Org Developer role as that may have a broader set of entitlements than what you may want to assign to non-admins. Same would apply to Legacy Org Admin once the new Org Admin [to be added soon] role is available. The legacy roles will be deprecated after the updated CockroachDB Cloud authorization model is generally available to all customers, and will be completely removed six months after that.
+
+### What are the legacy roles in the CockroachDB Cloud authorization model?
+
+Legacy Org Admin and Legacy Org Developer refer to the older roles available in a CockroachDB Cloud organization. These roles have a broad set of entitlements which make it hard to reason about least privilege principle in an organization. 
+
+The new incarnation for Legacy Org Admin will be Org Admin [to be added soon], while the same for Legacy Org Developer will be Org Member which will become the new default role. The new roles will have a limited set of entitlements so that it’s easier to reason about least privilege principle.
+
+The legacy roles will be deprecated after the updated CockroachDB Cloud authorization model is generally available to all customers, and will be completely removed six months after that. At general availability, it will no longer be possible to assign the legacy roles to human users or service accounts.
```

Comment:
Please use {{ site.data.products.db }} where possible. I'll skip correcting further instances. Whatever language you are using about deprecation, 6 months, etc, please use exactly the same language everywhere so that it's easier to maintain.